guide/getting-started/hardware.md

```diff
@@ -90,13 +94,19 @@ Running your [own node]({{ '/guide/glossary/node/#self-hosted-node' | relative_u
    height = 400
 %}
 
-Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
+It is quite common to have dedicated hardware to run Bitcoin [node software]({{ '/guide/getting-started/software/#nodes' | relative_url }}). Running a Bitcoin node on dedicated hardware ensures your node stays in sync with the network and is regularly verifying the network rules, increasing Bitcoin’s security. Some node software also comes packaged with other third-party applications which may benefit from dedicated hardware and more regular uptime. 
+
+Lightning node software can be run on a smartphone. Though, this often comes with trusting a third-party for certain node functions such as payment path construction. For this reason, dedicated hardware Lightning nodes may be a better option for those who do not want a trust-minimized setup. 
```

Comment:
> Lightning node software can be run on a smartphone.

This is a bold claim at this stage, is there a way to run a full lightning node on a smartphone? I don't think it is, though with LDK it should be possible. cc @moneyball ?

Reply:
It doesn't state "full node", just "node software" in general, and then explains some of the tradeoffs. Looks accurate to me.

Reply:
Node software assumes a node is run? You cannot run a node on a smartphone with any LN implementation afaik.

Reply:
You can run a Lightning node on a smartphone - Phoenix is an example: https://phoenix.acinq.co/faq#is-phoenix-a-real-lightning-node

Reply:
Interesting, wasn't aware of this. I guess eclair is the only implementation that makes this possible, will have to dig how they do it technically.